be/Cf.Contracts/Extensions/ServiceExtensions/ServiceExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Cf.Domain.Aggregates.Services;
+using Cf.Domain.Models;
+
+namespace Cf.Contracts.Extensions.ServiceExtensions;
+
+public static class ServiceExtensions
+{
+    public static Service UpdateFields(this Service service, ServiceAdditionalInfoModel additionalInfo)
+    {
+        service.ServiceName = additionalInfo.ServiceName != null ? additionalInfo.ServiceName : service.ServiceName;
+        service.Adress = additionalInfo.Address != null ? additionalInfo.Address : service.Adress;
```

Comment:
change all spelling from `adress` to `address` since request now demands `address`

![image](https://github.com/MatasAdmoneo/CarFans/assets/99315244/52c4c1ef-0d8c-4015-93d6-33da6991d61b)

Reply:
Good pick!